src/mono/wasi/runtime/driver.c

```diff
@@ -329,7 +329,33 @@ mono_wasm_register_bundled_satellite_assemblies (void)
 	}
 }
 
-void mono_wasm_link_icu_shim (void);
+void load_icu_data (void)
+{
+	FILE *fileptr;
+	unsigned char *buffer;
+	long filelen;
+	char filename[256];
+	sprintf(filename, "./icudt.dat");
+
+	fileptr = fopen(filename, "rb");
```

Comment:
`fopen("./icudt.dat", "rb");`

Reply:
filename is also used in lines 342 and 352.